components/automate-elasticsearch/habitat/plan.sh

```diff
@@ -49,9 +55,37 @@ do_build() {
 }
 
 do_install() {
-  cp -a "$(pkg_path_for ${vendor_origin}/elasticsearch)/es/"* "${pkg_prefix}/es/"
+  cd "$HAB_CACHE_SRC_PATH/elasticsearch-${pkg_version}"
+  install -vDm644 README.textile "${pkg_prefix}/README.textile"
+  install -vDm644 LICENSE.txt "${pkg_prefix}/LICENSE.txt"
+  install -vDm644 NOTICE.txt "${pkg_prefix}/NOTICE.txt"
+
+  # Elasticsearch is greedy when grabbing config files from /bin/..
+  # so we need to put the untemplated config dir out of reach
+  mkdir -p "${pkg_prefix}/es"
+  cp -a ./* "${pkg_prefix}/es"
+
+  # jvm.options needs to live relative to the binary.
+  # mkdir -p "$pkg_prefix/es/config"
+  # install -vDm644 config/jvm.options "$pkg_prefix/es/config/jvm.options"
+
+  # Delete unused binaries to save space
+  rm "${pkg_prefix}/es/bin/"*.bat "${pkg_prefix}/es/bin/"*.exe
+
+  LD_RUN_PATH=$LD_RUN_PATH:${pkg_prefix}/es/modules/x-pack-ml/platform/linux-x86_64/lib
+  export LD_RUN_PATH
+
+  _es_ml_bins=( "autoconfig" "autodetect" "categorize" "controller" "normalize" )
+  for bin in "${_es_ml_bins[@]}"; do
```

Comment:
If you prefer:

```suggestion
  for bin in autoconfig autodetect categorize controller normalize; do
```

but it doesn't matter.